doc/Language/control.pod6

```diff
@@ -24,7 +24,7 @@ also be written as:
 =head1 X<Blocks|control flow, blocks>
 
 Like many other languages, Raku uses C<blocks> enclosed by C<{> and C<}> to
-turn a sequence of statements into a single one. It is OK to skip the semicolon
+turn a sequence of statements into a single one. It is OK to ommit the semicolon
 between the last statement in a block and the closing C<}>.
```

Comment:
why the change? Skip is OK.

Reply:
For me, "skip" denotes that the semicolon is being passing over, however it doesn't suggest that it's being literally left out (as far the person writing is concerned) from the code as "omit" does. I might be wrong though so no biggie.  To top it up, I even misspelled the word. 😅

Reply:
Either one works, as long as the spelling is OK.